test/node_tests.jl

```diff
@@ -149,4 +158,25 @@ dz = node.test_msg.z - subnode.test_msg.z
 @test abs(dx) <= 1
 @test abs(dy) <= 2
 @test abs(dz) <= 3
-@show node.test_msg.x
\ No newline at end of file
+@show node.test_msg.x
+end
+
+## Test heartbeat
+@testset "Heartbeat" begin
```

Comment:
Here is the example of how to use the heartbeat. You have to enable it via the node options, and then it's as simple as creating a subscriber to the correct port.